### PR TITLE
SUBMARINE-538. Update the preparation content in release doc

### PR DESCRIPTION
**Pull request by jiwq**
### What is this PR for?
update the release steps

### What type of PR is it?
[Improvement]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-538

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: dev-support/cicd/HowToRelease.md [hunk 1]
```diff
@@ -6,20 +6,19 @@ If you have not already done so, generate your PGP key and append your [signing
...
-gpg --armor --export <your name> >> ~/key.txt
+
+# adding your key to the submarine KEYS
 svn co --depth immediates https://dist.apache.org/repos/dist apache-dist
 cd apache-dist
 svn update --set-depth infinity release/submarine
 cd release/submarine
-echo ~/key.txt >> KEYS
-svn add KEYS
+gpg --fingerprint <your name> >> KEYS
```
Comment (tangzhankun) [inline dev-support/cicd/HowToRelease.md:20]: @jiwq , one question, why change to "fingerprint"?

```
$ gpg --list-sigs ztang
pub   rsa2048 2019-05-29 [SC] [expires: 2021-05-28]
      4B3097ACE9D46117F0C4FC2BF82CBAEEEC87F784
uid           [ultimate] Zhankun Tang <ztang@apache.org>
sig 3        F82CBAEEEC87F784 2019-05-29  Zhankun Tang <ztang@apache.org>
sig          F4313315266526CB 2019-05-29  Sunil Govindan (CODE SIGNING KEY) <sunilg@apache.org>
sub   rsa2048 2019-05-29 [E] [expires: 2021-05-28]
sig          F82CBAEEEC87F784 2019-05-29  Zhankun Tang <ztang@apache.org>



$ gpg --fingerprint ztang
pub   rsa2048 2019-05-29 [SC] [expires: 2021-05-28]
      4B30 97AC E9D4 6117 F0C4  FC2B F82C BAEE EC87 F784
uid           [ultimate] Zhankun Tang <ztang@apache.org>
sub   rsa2048 2019-05-29 [E] [expires: 2021-05-28]
```

**Comment by tangzhankun** [comment]
@jiwq @pingsutw , one thing I missed in the document is the committer's key needs to be signed by a PMC whose key is already in the KEYS file.

**Comment by jiwq** [comment]
@tangzhankun Please help to review. I think we need the other PR to improve the docker images building.

**Comment by tangzhankun** [comment]
@jiwq Thanks for the contribution! It looks good to me. One question, do we also need to add the guide for releasing the Python SDK?

**Comment by jiwq** [comment]
> @jiwq Thanks for the contribution! It looks good to me. One question, do we also need to add the guide for releasing the Python SDK?

Yes. I'm working to develop the automatic release tool and write the docs.  Will commit if no more comments.

**Pull request closed without merging**